packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,180 @@
+import HomeIcon from "@mui/icons-material/Home";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import { styled } from "@mui/material/styles";
+import React, { ReactElement } from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import StyledTooltip from "./OSCALStyledTooltip";
+import {
+  OSCALSystemImplementationTableTitle,
+  StyledHeaderTableCell,
+  StyledTableRow,
+  StyledTableHead,
+} from "./OSCALSystemImplementationTableStyles";
+import { Property } from "@easydynamics/oscal-types";
+import { NIST_DEFAULT_NAMESPACE, namespaceOf } from "./oscal-utils/OSCALPropUtils";
+import { NotSpecifiedTypography } from "./StyledTypography";
+
+const OSCALPropertiesButton = styled(Button)(
+  ({ theme }) => `
+    color: ${theme.palette.primary.main};
+    margin-top: 1em;
+    margin-bottom: 0.5em;
+  `
+) as typeof Button;
+
+/**
+ * Helper to sort namespaces.
+ *
+ * @param propA The first property to compare
+ * @param propB The second property to compare
+ * @returns A positive number if A is before B alphabetically ot a negative number if B is before A
+ *   alphabetically.
+ */
+function byName(propA: Property, propB: Property): number {
+  return propA.name.localeCompare(propB.name);
+}
+
+interface OSCALPropertyProps {
+  /**
+   * Contains property elements
+   */
+  property: Property;
+}
+
+const OSCALProperty = (props: OSCALPropertyProps): ReactElement => {
+  const { property } = props;
```

Comment:
Can we update all the components in this file to do this?

```suggestion
const OSCALProperty: React.FC<OSCALPropertyProps> = ({ property }) => { 
```